packages/synchro-charts/src/testing/test-routes/widget-test-route.tsx

```diff
@@ -64,6 +73,8 @@ export class WidgetTestRoute {
       end: viewportEnd,
       duration,
       group: 'some-viewport-group',
+      yMin: Y_MIN,
```

Comment:
this will alter every single integration test - instead we should read these in as optional search query parameters and pass them into the test within the test file

Reply:
Done --- instead optional search query parameters.